build.gradle

```diff
@@ -405,44 +405,56 @@ tasks.register('cmakeJniLib', Exec) {
     standardOutput = outputStream
 }
 
+// Makes sure that `./gradlew clean` removes all JNI build artifacts
+tasks.clean.doFirst {
+    // Delete JNI build directory
+    delete "${projectDir}/jni/build"
+}
+
 task buildNmslib(type:Exec) {
     dependsOn cmakeJniLib
-    def makePath = findExecutable("make")
-    logger.lifecycle("Using make at: ${makePath}")
-    // Ensure makePath is treated as a String. If parsing to an int is required,
-    // handle it explicitly, though a path typically should not be parsed as an int.
-    if (makePath.isEmpty()) {
-        throw new GradleException("Make not found in PATH. Please install Make.")
+    def cmakePath = findExecutable("cmake")
+    logger.lifecycle("Using cmake at: ${cmakePath}")
```

Comment:
do we need this log?

Reply:
Not mandatory, but it helps when going over the gradle script execution to look for issues/failures.